packages/components/pie-webc-core/package.json

```diff
@@ -3,16 +3,20 @@
   "version": "0.12.0",
   "description": "PIE design system base classes, mixins and utilities for web components",
   "type": "module",
-  "main": "index.ts",
+  "main": "dist/index.js",
   "author": "Just Eat Takeaway.com - Design System Team",
   "license": "Apache-2.0",
   "scripts": {
+    "build": "run -T vite build",
     "lint:scripts": "run -T eslint .",
     "lint:scripts:fix": "run -T eslint . --fix",
     "test": "run -T vitest run",
     "test:ci": "yarn test",
     "test:watch": "run -T vitest"
   },
+  "dependencies": {
+    "lit": "3.1.0"
```

Comment:
shall we add `@lit/react` as well? 
I had a quick chat with @ashleynolan before and we believe it shouldn't affect the bundle size if users don't import the react wrapper and will eventually be three shaken,

Reply:
Looks like this will be non-trivial so I won't include it as part of this PR